src/course/create-course-dialog.tsx

```diff
@@ -33,7 +33,7 @@ export default function CreateCourseDialog() {
 
    return (
       <div>
-         <Button variant="contained" color="primary" onClick={handleClickOpen}>
+         <Button variant="contained" color="primary" onClick={handleClickOpen} data-testid="create-btn">
```

Comment:
Yessir, they're only used internally for tests, and are otherwise benign. They'll appear as data attributes in prod as well, but if that bothers you there's a Babel plugin to remove them in prod: https://www.npmjs.com/package/babel-plugin-jsx-remove-data-test-id